crates/sui-core/src/generate_format.rs

```diff
@@ -75,7 +75,7 @@ fn get_registry() -> Result<Registry> {
     tracer.trace_type::<MoveStructLayout>(&samples)?;
     tracer.trace_type::<MoveTypeLayout>(&samples)?;
     tracer.trace_type::<base_types::SuiAddress>(&samples)?;
-    tracer.trace_type::<UpdateItem>(&samples)?;
```

Comment:
Getting an error here that I do not quite understand `thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Custom("Failed to deserialize value: \"invalid length 0, expected an byte array of size 32\"")', crates/sui-core/src/generate_format.rs:104:35`